dbms/tests/msan_suppressions.txt

```diff
@@ -1,2 +1,10 @@
 # https://github.com/google/oss-fuzz/issues/1099
 fun:__gxx_personality_*
+
+# We apply std::tolower to uninitialized padding, but don't use the result, so
```

Comment:
Why don't just add suppression in source file via function attribute?
Look at `LZ4_decompress_faster.cpp`.

Reply:
Reached a conclusion that this is OK on Telegram.